hue/hue.sh

```diff
@@ -20,34 +20,36 @@ ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
 
 # Only run on the master node of the cluster
 if [[ "${ROLE}" == 'Master' ]]; then
+  # Install hue
   apt-get update
+  apt-get install -t jessie-backports hue-common -y
```

Comment:
Why not just install hue (with -t jessie-backports here) and remove the next line?

Reply:
done